.claude/skills/youtube-collector/scripts/register_channel.py

```
@@ -0,0 +1,247 @@
+
+
+def load_channels(channels_file: Path) -> dict:
+    """channels.yaml 로드"""
+    if not channels_file.exists():
+        return {'channels': []}
+
+    try:
+        with open(channels_file, 'r', encoding='utf-8') as f:
+            data = yaml.safe_load(f)
+            return data if data else {'channels': []}
+    except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)